plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckAnimatorManager.kt

```diff
@@ -117,9 +117,14 @@ internal class PuckAnimatorManager(
       accuracyCircleColor = settings2.accuracyRingColor
       accuracyCircleBorderColor = settings2.accuracyRingBorderColor
     }
-    bearingAnimator.apply {
-      enabled = settings2.puckBearingEnabled
-    }
+  }
+
+  internal fun setDisabled() {
+    bearingAnimator.enabled = false
+  }
+
+  internal fun setEnabled() {
+    bearingAnimator.enabled = true
```

Comment:
nit: maybe unify these two methods to `setEnabled(enabled: Boolean)`?

Reply:
I was going to do that but usually we have separate setters for each state, but since this is purely internal, maybe it is ok to unify